packages/theme/css/characteristics/content.css

```diff
@@ -0,0 +1,13 @@
+.salt-theme {
+  /* Colors */
+  --salt-content-foreground-highlight: var(--salt-palette-interact-background-active);
```

Comment:
would move this next to hover/active/visited  to make it more distinct between the variant tokens